source/clear-linux/get-started/bare-metal-install-server/bare-metal-install-server.rst

```diff
@@ -49,6 +49,11 @@ Follow these steps to install |CL| on the target system:
 #. Open the system BIOS setup menu by pressing the :kbd:`F2` key.
    Your BIOS setup menu entry point may vary.
 
+   .. note::
+      |CL| supports UEFI boot. Some hardware may list UEFI and non-UEFI USB
+      boot entries. In this case, you should select the ` -- UEFI` boot
+      sequence option.
+
```

Comment:
@mrkz  Per your request in #590 , I added this note. Will you please review asap. Let's try to keep this note simple.

Reply:
This is exactly what I was referring to, thanks for the note addition @mvincerx :tada: